federated_learning/financial_fraud_detection/README.md

```diff
@@ -80,6 +87,21 @@ cd tree-based/
 nvflare simulator jobs/creditcard_5_bagging_IID_split_uniform_lr -w ${PWD}/workspaces/xgboost_workspace_5_bagging_exponential_split_scaled_lr -n 5 -t 5
 ```
 
+You will see the execution as shown in the picture
```

Comment:
Please add a period in the end of the sentence.

Reply:
Done.